### PR TITLE
Build multiple go versions

### PR DESCRIPTION
**Pull request by ArangoGutierrez**

### CODE REVIEW

File: images/devel/Dockerfile [hunk 1]
```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-ARG GOLANG_VERSION=x.x.x
+ARG GOLANG_VERSION=x.x
```
Comment (elezar) [inline images/devel/Dockerfile:15]: Nit: Why the change?

File: .github/workflows/images.yaml [hunk 2]
```diff
@@ -31,9 +31,11 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        go-version: ["1.21","1.20","1.19"]
```
[the remark this answers is not in the document]
Comment (ArangoGutierrez) [reply .github/workflows/images.yaml:36]: this is an image for running CI tools, I don't see the advantage of a point release here. 
I agree production images for a specific component could be tied to a specific version, but for CI steps, the generic 1.x.latest should be ok

[the remark this answers is not in the document]
Comment (elezar) [reply .github/workflows/images.yaml:36]: Let us make this change at a later stage if required. For now, let's match what we currently do -- which is to use a point release.
